client/js/widgets/label.js

```diff
@@ -36,6 +68,12 @@ export class Label extends Widget {
         this.input.style.paddingTop = 'unset';
       }
     }
+    if(delta.html !== undefined) {
```

Comment:
You don't update the content if `delta.text` changes.

Reply:
I think you are saying that you would expect to see me to CALL setHTML or unsetHTML on text changes.  Yes, that sounds like the right way to do it.  I'm sure I did it a hacky way from within `setText` and `SET`.  I can work on fixing that, but it might take me a while. :)